src/components/capture/SaveButton.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button } from '@mui/material';
+import { createPublication } from 'api/publications';
+import { EditorStoreState } from 'components/editor/Store';
+import { buttonSx } from 'components/shared/Entity/Header';
+import { DraftSpecQuery } from 'hooks/useDraftSpecs';
+import { useRouteStore } from 'hooks/useRouteStore';
+import { useZustandStore } from 'hooks/useZustand';
+import { FormattedMessage } from 'react-intl';
+import { entityCreateStoreSelectors, FormStatus } from 'stores/Create';
+
+interface Props {
+    disabled: boolean;
+    onFailure: Function;
+    subscription: Function;
+}
+
+function CaptureSaveButton({ disabled, onFailure, subscription }: Props) {
```

Comment:
Creating a component for the button to house all the logic related to saving as well as rendering the component.